arb_os/gasAccounting.mini

```diff
@@ -165,6 +162,34 @@ public impure func gasAccounting_init() {    // initialize on boot, before we ha
     };
 }
 
+impure func setGasAccountingParamsFromSpeedLimit(speedLimitPerBlock: uint) {
+    gasAccountingParams = struct {
+        SpeedLimitPerBlock: speedLimitPerBlock,
+        GasPoolMax: int(speedLimitPerBlock * 5),
+        TxGasLimit: speedLimitPerBlock,
+    };
+}
+
+public impure func setGasAccountingParams(speedLimitPerBlock: uint, gasPoolMax: uint, txGasLimit: uint) {
```

Comment:
Just noting here I have mixed feelings about adjusting the parameters to make them sane vs returning an error if someone tries to set them to bad values. The main reason is because if the sanity checks fail, the inputs were likely produced in error and so they could be incorrect in other ways

Reply:
Changed this to reject bad parameters.  ArbOwner setGasAccountingParams call will revert if parameter calls are bad.